docs/en/users/04_Subscriptions.md

```diff
@@ -1,11 +1,31 @@
 # Adding a feed
 
-**TODO**
+ 1. To add a feed, copy the URL to RSS or Atom file's of the content wanted (for exemple, the framablog RSS URL is `https://framablog.org/feed/`).
+ 2. On FreshRSS, clic to “Subscriptions management”.
```

Comment:
```suggestion
 2. On FreshRSS, click “Subscriptions management”.
```

Reply:
(That should be subscription management really, but that's unrelated to this PR.)